TypeScript.py

```diff
@@ -8,6 +9,29 @@
 import sublime
 import sublime_plugin
 
+''' Enable logging '''
+logFileLevel = logging.DEBUG
+logConsLevel = logging.DEBUG
+
+def set_log_level(logger):
+    logger.logFile.setLevel(logFileLevel)
+    logger.console.setLevel(logConsLevel)
+
+# Sublime/Python 2 & 3 differ in the name of this module, thus package import
+# needs to be handled slightly differently
+if sys.version_info < (3, 0):
+    from libs import logger
+    set_log_level(logger)
+
+    # This package exposes the Sublime classes that need to be reexposed here
+    from libs import *
```

Comment:
You don't need to copy this bit (or the equivalent on line 31).  This is only here as in my original code I'm exporting the Sublime Commands classes from 'libs', which need to be reexported here.  You only need logger, which you imported in line 23.